101-event-grid-resource-events-to-webhook/README.md

```diff
@@ -0,0 +1,22 @@
+# Create Event Grid subscription for resource events
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-event-grid-resource-events-to-webhook%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template creates Event Grid subscription for either a resource group or Azure subscription. It sends the events to a WebHook.
+
+You need a WebHook endpoint for receiving the events. If you have one, pass that URI as the `endpoint` parameter. If you do not have an existing endpoint, the template in the **prereqs** folder deploys a web app that displays the event messages. Deploy this template to your subscription. Pass that URI in the format `https://<your-site>/api/updates/` For more information, see [Create a message endpoint](https://docs.microsoft.com/azure/event-grid/custom-event-quickstart#create-a-message-endpoint).
+
+To subscribe to events for an Azure subscription, deploy with the following command:
```

Comment:
remove instructions for deploying since they should be the same for every sample in the repo

Reply:
This template works for subscribing to either Azure subscription events or resource group events. The only difference is whether it is deployed to the subscription or resource group. In that case, aren't the instructions needed?